src/framework/accessibility/tests/accessibilitycontroller_tests.cpp

```
@@ -173,17 +177,17 @@ class Accessibility_ControllerTests : public ::testing::Test
 };
 
 TEST_F(Accessibility_ControllerTests, SendEventOnFocusChanged)
 {
-    //! [GIVEN] Accessibility is enabled
-    ON_CALL(*m_configuration, isAccessibleEnabled()).WillByDefault(Return(true));
-
     //! [GIVEN] Accessibility is active
-    ON_CALL(*m_configuration, isAccessibleActive()).WillByDefault(Return(true));
+    ON_CALL(*m_appRootObject, isAccessibilityActive()).WillByDefault(Return(true));
+    //! [GIVEN] Navigation has an active section (so isEnabled() returns true)
+    ON_CALL(*m_navigationController, activeSection()).WillByDefault(Return(reinterpret_cast<muse::ui::INavigationSection*>(1)));
```

Cppcheck (2.20.0)

[portability] Casting non-zero decimal integer literal to pointer.

(intToPointerCast)